### PR TITLE
Revitalize Storybook

### PR DESCRIPTION
**Pull request by Tropaul322**

### CODE REVIEW

File: template/apps/web/.storybook/preview.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import '@mantine/core/styles.css';
+
+import React, { useEffect } from 'react';
+import { addons } from '@storybook/preview-api';
+import { DARK_MODE_EVENT_NAME } from 'storybook-dark-mode';
+import {
+  MantineProvider,
+  useMantineColorScheme,
+} from '@mantine/core';
```
Comment (fruneen) [inline template/apps/web/.storybook/preview.tsx:9]:
```suggestion
import { MantineProvider, useMantineColorScheme } from '@mantine/core';
```

**Review by fruneen — CHANGES_REQUESTED** [review]
Good job! 👏

File: template/apps/web/.eslintrc.js [hunk 2]
```diff
@@ -10,8 +10,9 @@ module.exports = {
     'next',
     'airbnb',
     'airbnb-typescript',
+    'plugin:storybook/recommended'
   ],
-  ignorePatterns: ['.eslintrc.js'],
+  ignorePatterns: ['.eslintrc.js', '!.storybook'],
```
Comment (Tropaul322) [inline template/apps/web/.eslintrc.js:15]: I don't know how to do it differently, but I read that eslint automatically adds .storybook to ignore and it needs to be removed from ignore using !.

File: template/.github/workflows/storybook-deployment.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: yarn
+        # 👇 Adds Chromatic as a step in the workflow
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        # Chromatic GitHub Action options
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Tropaul322) [inline template/.github/workflows/storybook-deployment.yml:36]: If I'm not mistaken, GITHUB_TOKEN is automatically added by actions.

**Review by fruneen — APPROVED** [review]
Perfect!

**Pull request merged**